python_modules/dagster/dagster/_core/storage/alembic/versions/041_add_asset_check_evaluations_table.py

```diff
@@ -0,0 +1,69 @@
+"""add asset check evaluations table
+
+Revision ID: ec80dd91891a
+Revises: 5771160a95ad
+Create Date: 2023-08-21 12:05:47.817280
+
+"""
+import sqlalchemy as db
+from alembic import op
+from dagster._core.storage.migration.utils import has_index, has_table
+from dagster._core.storage.sql import get_current_timestamp
+from sqlalchemy.dialects import sqlite
+
+# revision identifiers, used by Alembic.
+revision = "ec80dd91891a"
+down_revision = "5771160a95ad"
+branch_labels = None
+depends_on = None
+
+TABLE_NAME = "asset_check_executions"
+INDEX_NAME = "idx_asset_check_executions_unique"
+
+
+def upgrade():
+    if not has_table(TABLE_NAME):
+        op.create_table(
+            TABLE_NAME,
+            db.Column(
+                "id",
+                db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+                primary_key=True,
+                autoincrement=True,
+            ),
+            db.Column("asset_key", db.Text),
+            db.Column("check_name", db.Text),
+            db.Column(
+                "partition", db.Text
+            ),  # Currently unused. Planned for future partition support
+            db.Column("run_id", db.Text),
+            db.Column("execution_status", db.Text),  # Planned, Success, or Failure
```

Comment:
if the event_record is a planned event, then the status will be planned. If it's an evaluation event, then it will be the pass/fail result of the evaluation